ql/src/test/org/apache/hadoop/hive/ql/TxnCommandsBaseForTests.java

```diff
@@ -246,6 +246,13 @@ public static void runInitiator(HiveConf hiveConf) throws Exception {
   public static void runWorker(HiveConf hiveConf) throws Exception {
     runCompactorThread(hiveConf, CompactorThreadType.WORKER);
   }
+  public static void runWorker(HiveConf hiveConf, String poolName) throws Exception {
```

Comment:
why do you need this method. can we add  
````
((Worker)t).setPoolName(poolName);
````
in `runCompactorThread` and keep the original signature with `default` value?

Reply:
Done